README.md

```diff
@@ -0,0 +1,101 @@
+# Releasing New Documentation Version
+
+## Adding docs for a new release
+
+Official docs for a release are built and pushed when a final tag
+is created via GitHub. When -rc tags are built, you should see a directory
+named after the tag with the -rc suffix and the prefix "v"
+truncated. For example, v0.1.0-rc1 would become just 0.1. This way
+you can preview your documentation before the release.
+
+After the final tag is created, an action will move all the docs from
+`release/xx` branch to a directory named after your release version.
+
+When an -rc tag dir is pushed to the gh-pages branch, update versions.html as follows:
```

Comment:
To make it more clear that an action won't do this, since the previous paragraph talked about actions. Also making it more clear that this isn't necessary every time a -rc tag is created, only the first one for a given release.

```suggestion
When the first -rc tag dir for a new release branch is pushed to the gh-pages
branch, you will need to manually update versions.html as follows:
```